.github/workflows/report_per_repo.yml

```diff
@@ -46,8 +46,10 @@ jobs:
             sw-name: starter-kit-lsaai
           - repository: neicnordic/sensitive-data-archive
             sw-name: neicnordic/sensitive-data-archive
-          - repository: NBISweden/sda-cli
```

Comment:
You accidentally removed the repo `NBISweden/sda-cli`. Can you please add it back?